vyper/opcodes.py

```diff
@@ -48,6 +48,7 @@
     'NUMBER': [0x43, 0, 1, 2],
     'DIFFICULTY': [0x44, 0, 1, 2],
     'GASLIMIT': [0x45, 0, 1, 2],
+    'CHAINID': [0x46, 0, 1, 2],
```

Comment:
Can you add a comment like TODO RULESET so that when we go back and implement ruleset versioning it's easy to find

Reply:
```suggestion
    'CHAINID': [0x46, 0, 1, 2],  # Added Istanbul Ruleset
```